demo/mindstorms_widgets.py

```diff
@@ -0,0 +1,321 @@
+                     seconds=0, degrees=0, rotations=0,
+                     invert=False, regulated=True, brake_at_end=True ):
+        self._run_motor( self.motor[motor], mode, power, 
+                         seconds, degrees, rotations,
+                         invert, regulated, brake_at_end )
+
+    def medium_motor( self, mode, power=50,  
+                      seconds=0, degrees=0, rotations=0,
+                      invert=False, regulated=True, brake_at_end=True ):
+        self._run_motor( self.motor['medium'], mode, power, 
+                         seconds, degrees, rotations, 
+                         invert, regulated, brake_at_end )
+
+    def move_steering( self, mode, direction=0, power=50, 
+                       seconds=0, degrees=0, rotations=0,
+                       invert=False, regulated=True, brake_at_end=True ):
+        '''
+        direction [-100, 100]:
+        * -100 = pivot left i.e. right motor = power
+        *                        left motor  = -power
+        * -50  = sharp left i.e. right motor = power
+        *                        left motor  = 0
+        * -25  = turn  left i.e. right motor = power
+        *                        left motor  = 1/2 power
+        * 0    = straight
+        * 25   = turn right
+        * 50   = sharp right
+        * 100  = pivot right
+        '''
+        self._validate_inputs( degrees, rotations )
+        powers = steering( direction, power )
+        self._set_motor_attribs( 'large', 
+                                 [self.motor['left'], self.motor['right']], 
+                                 powers, invert, regulated, brake_at_end )
+        self._move( mode, seconds, degrees, rotations )
+        
+    def move_tank( self, mode, direction=0, lr_power=None,
+                   seconds=0, degrees=0, rotations=0,
+                   invert=False, regulated=True, brake_at_end=True ):
+        self._validate_inputs( degrees, rotations )
+        if not lr_power:
+            lr_power=[50,50]
+        self._set_motor_attribs( 'large',
+                                 [self.motor['left'], self.motor['right']],
+                                 lr_power, invert, regulated, brake_at_end )
+        self._move( mode, seconds, degrees, rotations )
+
+    def brick_status_light( self, mode, color='green', pulse=False ):
+        if mode == 'on':
+            if color == 'green':
+                led.green_on()
+                led.red_off()
+            elif color == 'orange':
+                led.all_on()
+            elif color == 'red':
+                led.green_off()
+                led.red_on()
+        elif mode == 'off':
+            led.all_off()
+        elif mode == 'reset':
+            led.all_off()
+        if mode == 'on' and pulse:
+            #led.red_left.flash( 200 )
+            #todo:Seems that the delay_on/delay_off attributes aren't there in
+            #/sys/class/leds/ev3-left0:red:ev3dev/
+            pass

```

Comment:
`flash()` should be fixed in the latest version (see ev3dev/ev3dev-lang#92). Forget my comment about setting the trigger to `timer` manually, since flash does that anyway.